.doc_gen/metadata/dynamodb_metadata.yaml

```diff
@@ -1,4 +1,30 @@
 # zexi 0.4.0
+dynamodb_ExecuteStatement:
+  title: Query a &DDB; table using PartiQL and &AWS; SDK 

```

Comment:
Should this sentence end with “and an &AWS; SDK” or “and the &AWS; SDK”?

Reply:
fixed